src/Microsoft.Windows.CsWin32/Generator.cs

```diff
@@ -1667,6 +1678,24 @@ protected virtual void Dispose(bool disposing)
             }
         }
 
+        private static ClassDeclarationSyntax ClassDeclarationWithTrivia(string name) => ClassDeclarationWithTrivia(Identifier(name));
+
+        private static ClassDeclarationSyntax ClassDeclarationWithTrivia(SyntaxToken name) => ClassDeclaration(name)
+            .WithKeyword(TokenWithSpace(SyntaxKind.ClassKeyword))
+            .WithOpenBraceToken(Token(TriviaList(LineFeed), SyntaxKind.OpenBraceToken, TriviaList(LineFeed)))
+            .WithCloseBraceToken(TokenWithLineFeed(SyntaxKind.CloseBraceToken));
+
+        private static UsingDirectiveSyntax UsingDirectiveWithTrivia(NameSyntax name, NameEqualsSyntax? alias = null) => UsingDirective(
+            usingKeyword: Token(TriviaList(ElasticTab), SyntaxKind.UsingKeyword, TriviaList(ElasticSpace)),
+            staticKeyword: default,
+            alias: alias?.WithEqualsToken(Token(TriviaList(ElasticSpace), SyntaxKind.EqualsToken, TriviaList(ElasticSpace))),
+            name: name,
+            semicolonToken: TokenWithLineFeed(SyntaxKind.SemicolonToken));
+
+        private static SyntaxToken TokenWithSpace(SyntaxKind syntaxKind) => Token(TriviaList(), syntaxKind, TriviaList(ElasticSpace));
```

Comment:
💡 Avoid elastic for this. It's a hint to the formatter but unnecessary for generation that does not go through the formatter on the way to a string.